pontoon/base/models.py

```diff
@@ -1335,6 +1340,12 @@ class Entity(DirtyFieldsMixin, models.Model):
     comment = models.TextField(blank=True)
     order = models.PositiveIntegerField(default=0)
     source = JSONField(blank=True, default=list)  # List of paths to source code files
+
+    # extra stores data that we want to save for the specific format
+    # this entity is stored in, but that we otherwise don't care
+    # about.
+    extra = JSONField(default=extra_default)
```

Comment:
Makes sense.

@Osmose Do you remember why didn't use simply use {} here?
https://github.com/mozilla/pontoon/blame/master/pontoon/base/models.py#L1521